x/gamm/twap/store_test.go

```diff
@@ -0,0 +1,181 @@
+package twap_test
+
+import (
+	"time"
+
+	"github.com/osmosis-labs/osmosis/v10/x/gamm/twap/types"
+)
+
```

Comment:
This isn't right though, It should only appear once per module, otherwise all tests run twice, this was an issue with mint as well, that was fixed in #2137

Reply:
The way you do this without running tests multiple times is by adding the following instead:

`var _ = suite.TestingSuite(nil)`

Reply:
ahh didn't realize that

@czarcas7ic this doesn't work for me. How did you get it to work?

Reply:
What I do is I have vscode run all tests in a package on save